fastlane/lanes/localization.rb

```diff
@@ -126,26 +124,59 @@
   lane :generate_strings_file_for_glotpress do |options|
     cocoapods
 
-    wordpress_en_lproj = File.join('WordPress', 'Resources', 'en.lproj')
-    ios_generate_strings_file_from_code(
-      paths: ['WordPress/', 'Pods/WordPress*/', 'Pods/WPMediaPicker/', 'WordPressShared/WordPressShared/', 'Pods/Gutenberg/'],
-      exclude: ['*Vendor*', 'WordPress/WordPressTest/**', '**/AppLocalizedString.swift'],
-      routines: ['AppLocalizedString'],
-      output_dir: wordpress_en_lproj
-    )
-
-    # Merge various manually-maintained `.strings` files into the previously generated `Localizable.strings` so their extra keys are also imported in GlotPress.
-    # Note: We will re-extract the translations back during `download_localized_strings_and_metadata` (via a call to `ios_extract_keys_from_strings_files`)
-    ios_merge_strings_files(
-      paths_to_merge: MANUALLY_MAINTAINED_STRINGS_FILES,
-      destination: File.join(wordpress_en_lproj, 'Localizable.strings')
-    )
+    # On top of fetching the latest Pods, we also need to fetch the source for the Gutenberg code.
+    # To get it, we need to manually clone the repo, since Gutenberg is distributed via XCFramework.
+    # XCFrameworks are binary targets and cannot extract strings via genstrings from there.
+    require_relative './../../Gutenberg/version'
+
+    ref = GUTENBERG_CONFIG[:tag] || GUTENBERG_CONFIG[:commit]
+
+    UI.user_error!('Could not find Gutenberg ref to clone the repository in order to access its strings.') if ref.nil?
+
+    UI.user_error!('Could not find GitHub organization name to clone Gutenberg in order to access its strings.') unless defined?(GITHUB_ORG)
+
+    UI.user_error!('Could not find GitHub repository name to clone Gutenberg in order to access its strings.') unless defined?(REPO_NAME)
+
+    # Create a temporary directory to clone Gutenberg into.
+    # We'll run the rest of the automation from within the block, but notice that only the Gutenbreg cloning happens within the temporary directory.
+    gutenberg_clone_name = 'Gutenberg-Strings-Clone'
+    Dir.mktmpdir do |tempdir|
+      Dir.chdir(tempdir) do
+        repo_url = "https://github.com/#{GITHUB_ORG}/#{REPO_NAME}"
+        UI.message("Cloning Gutenberg from #{repo_url} into #{gutenberg_clone_name}. This might take a few minutes…")
+        sh("git clone --depth 1 #{repo_url} #{gutenberg_clone_name}")
+        Dir.chdir(gutenberg_clone_name) do
+          if GUTENBERG_CONFIG[:tag]
+            sh("git fetch origin refs/tags/#{GUTENBERG_CONFIG[:tag]}:refs/tags/#{GUTENBERG_CONFIG[:tag]}")
+            sh("git checkout refs/tags/#{GUTENBERG_CONFIG[:tag]}")
+          else
+            sh("git fetch origin #{ref}")
+            sh("git checkout #{ref}")
```

Comment:
I only now realize that I missed a chance to use Ruby's Git utilities, which would probably be better than shelling out like this.

I don't think it's worth delaying a merge, particularly because of the translation fix. I shall follow up.